agent-control/src/cli/on_host/config_gen/config.rs

```diff
@@ -149,53 +115,36 @@ mod tests {
         assert_eq!(result, expected_map);
     }
 
-    #[test]
-    fn test_serialize_proxy_config_all_empty_options() {
-        let proxy_config = ProxyConfig {
-            proxy_url: Some(String::new()),
-            proxy_ca_bundle_dir: Some(String::new()),
-            proxy_ca_bundle_file: Some(String::new()),
-            ignore_system_proxy: false,
-        };
-
-        let serialized = serde_yaml::to_string(&proxy_config).unwrap();
-        // Only ignore_system_proxy should be present
-        assert_eq!(serialized.trim(), "ignore_system_proxy: false");
-    }
-
-    #[test]
-    fn test_serialize_proxy_config_none_options() {
-        let proxy_config = ProxyConfig {
-            proxy_url: None,
-            proxy_ca_bundle_dir: None,
-            proxy_ca_bundle_file: None,
-            ignore_system_proxy: true,
-        };
-
-        let serialized = serde_yaml::to_string(&proxy_config).unwrap();
-        // Only ignore_system_proxy should be present
-        assert_eq!(serialized.trim(), "ignore_system_proxy: true");
-    }
-
     #[rstest]
-    #[case("", ProxyConfig::default())]
     #[case(
-        "--proxy-url https://proxy.url --proxy-ca-bundle-dir=/bundle/dir --proxy-ca-bundle-file=/bundle/file --ignore-system-proxy true",
-        ProxyConfig{proxy_url: Some("https://proxy.url".into()), proxy_ca_bundle_dir: Some("/bundle/dir".into()), proxy_ca_bundle_file: Some("/bundle/file".into()), ignore_system_proxy: true},
+        Config {
+            fleet_control: None,
+            server: Server { enabled: true },
+            proxy: None,
+            agents: HashMap::new(),
+        },
+        r#"{"server":{"enabled":true},"agents":{}}"#
+    )]
+    #[case(
```

Comment:
nit:  add the case name so is easy to understand what is being tested

Reply:
Names added here: [17aaa2a](https://github.com/newrelic/newrelic-agent-control/pull/1919/commits/17aaa2a0f596b7c472f55787b8bf4b9e2f0a93f8)